Chapter4_TheGreatestTheoremNeverTold/Ch4_LawOfLargeNumbers_PyMC_current.ipynb

```diff
@@ -0,0 +1,1487 @@
+{
```

Comment:
<p>I don't think you have to pass tune, and does it reuqire that many samples?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CamDavidsonPilon/Probabilistic-Programming-and-Bayesian-Methods-for-Hackers/pull/548/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>I'm not quite sure about the sample size, </p><p>because according to the PYMC2 and PYMC3 version, it samples one chain on 20000 samples and burned the 1/4 of 20000 (5000).</p><p> </p><p>I try to keep the code minimal modified with PYMC2 and PYMC3 version, maybe "15000 sample/ one chain/5000 tune" is more appropriate?</p><p> </p>

Reply:
<p>You can also just leave it.</p>